scripts/UnitTests/Backend.Tests.ps1

```diff
@@ -1,7 +1,6 @@
 $here = Split-Path -Parent $MyInvocation.MyCommand.Path
 $parentPath = Split-Path -Parent $here
 . "$parentPath\Backend.ps1"
-
```

Comment:
do not remove the line - there always should be a line or two to separate import code from the actual code